src/test/java/ci/PersistentLogsTest.java

```
@@ -30,10 +30,14 @@ void test_add_log_and_check_state() {
         }
         PersistentLogs test_logs = new PersistentLogs("test_logs");
         Log_entry le = new Log_entry(Log_entry.Log_type.PUSH, "refs/heads/master", "this_is_a_test_commit", new Date(), Log_entry.Test_status.SUCCESS, "this is a test gradle output");
-        int num_logs = test_logs.build_number; //all_logs().length;
-        test_logs.add_log(le);
-        assertEquals(num_logs + 1, test_logs.build_number);
-        assertEquals(true, le.toString().equals(PersistentLogs.get_log(test_logs.all_logs()[num_logs]).toString()));
+        int num_logs = test_logs.build_number;
+        try {
+            test_logs.add_log(le);
+            assertEquals(num_logs + 1, test_logs.build_number);
+            assertEquals(true, le.toString().equals(PersistentLogs.get_log(test_logs.all_logs()[num_logs]).toString()));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Why do we silently fail if there's an exception? Wouldn't it be better if the test failed if something goes wrong (ie. don't catch the exception)?